schemas/zowe-yaml-schema.json

```diff
@@ -58,8 +58,16 @@
                   "default": "<hlq>.SZWEAUTH"
                 },
                 "authPluginLib": {
-                  "$ref": "/schemas/v2/server-common#zoweDataset",
-                  "description": "APF authorized LOADLIB for Zowe ZIS Plugins"
+                  "oneOf": [{
```

Comment:
I am not sure about this, `authPluginLib` is not defined in defaults and it is not required by schema, user has already the options:
* Either specify dataset - which must be valid from schema (regex) point of view
* Omit this completely as intended not to use it 

Allowing empty dataset name is strange solution.

Reply:
Yeah, I don't love this either and I'm going to revert it. An empty value could also imply behaviors other than an effectively unset variable.

I was working from the perspective of a user who starts with an example-zowe.yaml, and as they go through the yaml file field-by-field they blank the authPluginLib since they don't want or need it. I think this user scenario could happen, and with the reverted behavior, the schema validator will tell them they need a dataset value, which they then fill out, which enables authPluginLib when they don't want it. All this assumes they've gone off-doc at least a little bit, which also seems plausible to me :). I would like to revisit the schema for cases like this another time...